test/over_react/util/map_util_test.dart

```diff
@@ -253,5 +254,54 @@ main() {
         expect(newStyleFromProps(domProps()), equals({}));
       });
     });
+
+    group('getBackingMap()', () {
+      group('when passed in a props object', () {
+        JsBackedMap testMap = JsBackedMap.from({'id': 'test'});
+
+        var props = getBackingMap(TestProps(testMap));
+
+        test('returns a map', () {
+          expect(props, TypeMatcher<Map>());
```

Comment:
I think we should verify here that the map instance we expect is returned. So we could do:
```dart
expect(props, same(testMap));
```